packages/css/src/plugins/__tests__/atomicify-rules.test.tsx

```diff
@@ -200,14 +211,12 @@ describe('atomicify rules', () => {
       }
     `;
     const secondActual = transform`
-      :first-child {
+      &:first-child {
         color: blue;
       }
     `;
 
-    const expected = '._roi113q2:first-child{color:blue}';
-    expect(firstActual).toEqual(expected);
-    expect(secondActual).toEqual(expected);
+    expect(firstActual).toEqual(secondActual);
```

Comment:
this test now is irrelevant because the nesting selector is added in the dangling pseudo plugin - but i left the test for... whatever reason...